backend/graphql/resolvers/courseResolvers.ts

```diff
@@ -18,6 +22,12 @@ import { Role } from "../../types";
 import { CourseVisibilityAttributes } from "../../models/course.model";
 import { assertNever } from "../../utilities/errorUtils";
 
+const defaultBucket = process.env.FIREBASE_STORAGE_DEFAULT_BUCKET || "";
+const fileStorageService = new FileStorageService(defaultBucket);
```

Comment:
what would happen here if there's no env variable for the default bucket? Would you still be able to create a FileStorageService with a bucket of ""?

Reply:
You're right that this wouldn't work in that case. However, since we're in control of the environment, I don't think this case would ever occur.

Reply:
I see. I guess no need for the || "" in that case, but no harm in leaving it in.